### PR TITLE
Olympus.Olympus version 21.01.11.07

### PR DESCRIPTION
**Pull request by lemonjacob**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95906)

### CODE REVIEW

**Comment by Trenly** [comment]
@lemonjacob - I'm not sure what happened, but anytime I try to check out the commit to test it, I get `fatal: reference is not a tree: 028644f4fa8de97784ce4022c5af7cf9cd8adbf2`. Is there any way you could try re-pushing to the branch?

File: manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Olympus.Olympus
```
Comment (Trenly) [inline manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.installer.yaml:4]:
```suggestion
PackageIdentifier: EverestAPI.Olympus
```

File: manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Olympus.Olympus
```
Comment (Trenly) [inline manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: EverestAPI.Olympus
```

File: manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Olympus.Olympus
```
Comment (Trenly) [inline manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.yaml:4]:
```suggestion
PackageIdentifier: EverestAPI.Olympus
```

File: manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Olympus.Olympus
+PackageVersion: 21.01.11.07
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://github.com/EverestAPI/Olympus/releases/download/v21.01.11.07/Olympus.exe
+  InstallerSha256: 846794AB91F137EA04C1C530069C9AC992EC43DFBFCAFC3317773225A19A0689
+  Dependencies:
```
Comment (Trenly) [inline manifests/o/Olympus/Olympus/21.01.11.07/Olympus.Olympus.installer.yaml:11]: Dependencies currently aren't supported.

**Comment by lemonjacob** [comment]
> @lemonjacob - I'm not sure what happened, but anytime I try to check out the commit to test it, I get `fatal: reference is not a tree: 028644f4fa8de97784ce4022c5af7cf9cd8adbf2`. Is there any way you could try re-pushing to the branch?

I think I fixed it

**Comment by Trenly** [comment]
> > @lemonjacob - I'm not sure what happened, but anytime I try to check out the commit to test it, I get `fatal: reference is not a tree: 028644f4fa8de97784ce4022c5af7cf9cd8adbf2`. Is there any way you could try re-pushing to the branch?
> 
> I think I fixed it

Thanks! Looks like git is detecting it as a valid tree now, will test!

**Review by Trenly — CHANGES_REQUESTED** [review]
[Policy] Dependencies
@Stephengillie / @denelon - Hardware too please

image

**Comment by Trenly** [comment]
[Policy] Dependencies

**Pull request closed without merging**